rubycritic.gemspec

```
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.summary       = 'RubyCritic is a Ruby code quality reporter'
   spec.homepage      = 'https://github.com/whitesmith/rubycritic'
   spec.license       = 'MIT'
-  spec.required_ruby_version = '>= 2.4.0'
+  spec.required_ruby_version = '> 2.4.0'
```

Let's make it like so:

```suggestion
  spec.required_ruby_version = '>= 2.5.0'
```

I'm worried that with `'> 2.4.0'` alone, versions like `2.4.1` would be allowed

---

Yes absolutely. Its a miss from my end. It should be what you suggest. will make a change and push.

---

Hey @nunosilva800 , I have pushed the change. Thanks for the suggestion.